fractions/microprofile/microprofile-jwt/src/main/java/org/wildfly/swarm/microprofile/jwtauth/deployment/auth/config/JWTAuthContextInfoProvider.java

```diff
@@ -80,6 +80,8 @@
                 NONE.equals(super.getMpJwtPublicKey().get()) && NONE.equals(super.getMpJwtLocation().get())) {
             return Optional.empty();
         }
+        log.warn("The use of all mpjwt.* properties is deprecated, use thorntail.microprofile.jwt.* properties instead");
```

Comment:
I think the MP JWT spec defines some of these properties, no? Hmm, actually no, the MP JWT spec defines a couple of `mp.jwt.verify.*` properties. Where do we handle those?

Reply:
In smallrye-jwt, in the base provider.

Reply:
Ah sure, that should have been obvious :-) Thanks!